packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -243,10 +244,19 @@ function WorklistDataGrid(props) {
   // callback Hooks
 
   const gridRef = useCallback((node) => {
+    if (!node) {
+      return;
+    }
+
     grid.current = node;
 
-    // Update table height state variable
-    setTableHeight(grid.current.offsetHeight - 1);
+    const resizeObserver = new ResizeObserver(() => {
+      // Update table height state variable
+      setTableHeight(grid.current.offsetHeight - 1);
```

Comment:
Is it possible to add a WDIO test where the table height changes?

Reply:
I am not aware of a way to do that.  Do we have an example of re-rendering a component with different prop values?

Reply:
Won't the component automatically rerender if we update the input data?